packages/app-bridge/src/react/useBlockTemplates.ts

```diff
@@ -10,6 +10,7 @@ export const useBlockTemplates = (appBridge: AppBridgeBlock) => {
     const blockId = appBridge.getBlockId();
 
     const [blockTemplates, setBlockTemplates] = useState<Record<string, Template[]>>({});
+    const [errorMessage, setErrorMessage] = useState<string>('');
```

Comment:
As you added logic, please make sure to cover it with tests

Reply:
Sure, just slipped out of my mind. 
Fixed the existing error-related test case and added a new one which is specific to error handling.